scripts/run_integration_tests.sh

```diff
@@ -67,10 +71,11 @@ if [ "$UPDATE_SNAPSHOTS" = "true" ]; then
             echo "Error: Cannot update snapshot files directly through the master branch, please create a development branch then run the script again."
             exit 1            
         else
-            echo "Commiting and pushing up snapshot changes to ${BRANCH}. Please create a pull request on GitHub."
+            echo "Commiting and pushing up snapshot changes to ${BRANCH}."
             git add .
             git commit -m "Update snapshots for integration tests"
             git push origin $BRANCH
+            echo "Please create a pull request on GitHub."
```

Comment:
Unsure about this, I was stuck between echoing a reminder or having the github stuff automated

Reply:
I'm not sure we want to automatically commit and push the snapshot changes every time we run the integration tests -- our other libraries don't do that. What's the rationale for doing so?

Reply:
Honestly my reasoning was just so people wouldn't forget to commit and push but I feel like if someone wanted to update the snapshots they should probably remember to do so.

Reply:
I removed the git stuff and just left a echo reminder